lib/web/apiserver.go

```diff
@@ -769,6 +769,7 @@ func (h *Handler) bindDefaultEndpoints() {
 	h.GET("/webapi/auth/export", h.authExportPublic)
 
 	// join token handlers
+	h.POST("/webapi/token/yaml", h.WithAuth(h.upsertTokenContent))
```

Comment:
If this is an upsert, would PUT be more accurate method?

Reply:
Yeah you're right. I'll update